bin/app.ts

```diff
@@ -149,6 +153,8 @@ export class APIPipeline extends Stack {
         SYNTH_SWITCH_API_KEY: syntheticSwitchApiKeySecret.secretValueFromJson('AUTH_BETA').toString(),
         ROUTING_API_URL: urlSecrets.secretValueFromJson('ROUTING_API_BETA').toString(),
         SERVICE_URL: urlSecrets.secretValueFromJson('GOUDA_SERVICE_BETA').toString(),
+        PORTION_API_URL: urlSecrets.secretValueFromJson('PORTION_API_BETA').toString(),
+        PORTION_FLAG: portionFlagSecret.secretValueFromJson('PORTION_FLAG').toString(),
```

Comment:
if this flag is just for enabling / disabling then I'd prefer calling it `ENABLE_PORTION`